### PR TITLE
Add a link to a existing document

### PR DESCRIPTION
**Pull request by pollirrata**
Link to existing tutorial page to help on making the point clearer.

### CODE REVIEW

File: entity-framework/efcore-and-ef6/porting/port-edmx.md [hunk 1]
```diff
@@ -44,3 +44,6 @@ At this point, it's a matter of addressing compilation errors and reviewing code
 ## Test the port
 
 Just because your application compiles, does not mean it is successfully ported to EF Core. You will need to test all areas of your application to ensure that none of the behavior changes have adversely impacted your application.
+
+## More examples
+For an additional reference on how to work with an existing database, see ["Getting Started with EF Core on ASP.NET Core with an Existing Database"](https://docs.microsoft.com/en-us/ef/core/get-started/aspnetcore/existing-db).
```
Comment (divega) [inline entity-framework/efcore-and-ef6/porting/port-edmx.md]: Thanks for the PR. Here is some feedback. BTW, the reason I give you this feedback is that since you have sent a couple useful improvements our way, you may very well be interested in learning how to make your PRs better and ready to merge. That way you could help even more, but if this is too much of a burden I can go ahead and merge your PR as is and then make the changes myself.

- Please remove the new title and turn the added text into a TIP (you can search this repo to find examples of how those are formatted)
- Please make the URL relative (you can find about that too in other pages) and remove the locale (en-us) from it
- In the future it helps if you create your PR based on mater (our staging branch) rather than our live branch.

**Comment by Rick-Anderson** [comment]
@divega  I made the changes. XREF is much better than relative links for the following reasons:

* Better error reporting.
* You can copy/paste them anywhere (they're not relative).

You create them using the UID of the topic.

I also changed the wording per the MS style guide.

**Comment by divega** [comment]
Thanks @pollirrata and @Rick-Anderson!
